sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+Thus, the rollout for sBTC will include two key phases:
+
+**Bootstrap:** In this phase an initial group of Signers selected through an open community governance process will be responsible for Signing sBTC transactions on the network. The initial group of Signers will be unchanged throughout the Bootstrap phase of the release schedule. During this phase, sBTC will have a distinct signer set (known as the “sBTC Bootstrap Signer”). And sBTC will not explicitly be integrated into the Stacks Proof of Transfer consensus. As a result, this release can activate without a hard fork.
+
+**Signer Rotation:** In this phase the full design of the sBTC system with an open and decentralized Signer set is reached, increasing decentralization after the system is further tested in the bootstrap phase. At this point, sBTC is integrated into the Proof of Transfer consensus and Signers receive rewards for fulfilling sBTC operations. This release does require a hard fork.
+
+#### Differences in the sBTC Bootstrap Phase:
+- sBTC Signer eligibility criteria will be confirmed through community governance vote. The eligibility criteria to become an sBTC signer is described below and will take into account Signer performance and availability. sBTC Signers that meet the criteria are confirmed by the sBTC Working Group. A subsequent release will implement an open and dynamically rotating signer set for each Proof of Transfer cycle, as described in the original sBTC design documents.
+- sBTC Bootstrap Signers are separate from Stacks Signers. In a subsequent sBTC release, and accompanying Stacks network hard fork, Stacks signers will be required to become sBTC signers. The bootstrapping period uses a separate subset of Stacks signers to secure the sBTC protocol and Signer operations are not explicitly linked with slashing of rewards.
+- sBTC deposits will be triggered via an API call. During the bootstrap phase sBTC deposits must be initiated via an API call to alert the signers to the presence of a deposit. After the bootstrapping phase users will have the option to alert the Signers of a deposit via a Stacks contract call, but the API will remain as a fee-less option to communicate with the Signers.
+
+These changes enable sBTC to be launched earlier and with less risk than the long term decentralized approach. The sBTC release with an open and dynamic signer set is expected to be released early 2025, which would enshrine the sBTC protocol into Stacks consensus.
+
+## Specification
+
+### Deposits (Minting sBTC)
+An API call referencing a Bitcoin transaction initiates the deposit process that can be completed within a single Bitcoin block. The API is separate from the sBTC Bootstrap Signers but will need access to the sBTC Bootstrap Signers to let them know which transactions to pick up.
```

Comment:
A few questions:

* Who runs this API?

* Does this API need to talk to all the signers, or will just one signer suffice?

* Can signers themselves provide API endpoints?

* What happens (who's responsible) when the API service can't talk to the signers?

* Can signers talk to each other out-of-band to figure out when to accept a peg-in?  i.e. if signer A learns of a peg-in, can they forward it to signer B?  If so, then it would be sufficient for the user to contact any honest signer to initiate a peg-in, and there wouldn't need to be a whole other API service.

Reply:
a single bitcoin block makes me nervous since we've seen an abnormal number of 1 block reorgs in the recent past. 
is there anything in this specification that accounts for that? will the tx be resent?